src/api/ApiManager.ts

```diff
@@ -29,6 +29,12 @@ export default class ApiManager {
         })
     }
 
+    callApi(path: string, method: 'GET' | 'POST' | 'POST_DATA', data: any) {
```

Comment:
Yes, it would be much complex to create the ReadStream needed by the only API that use it (`uploadAppData`). Also it is already covered by `deploy` command.
We could delete `POST_DATA` from that function signature.